nix/nix.go

```diff
@@ -176,6 +176,24 @@ func (ctx *NixContext) GetBuildShell(deploymentPath string) (buildShell *string,
 	return buildShell, nil
 }
 
+func (ctx *NixContext) EvalHosts(deploymentPath string, attribute string) {
+	args := []string{ctx.EvalMachines,
+		"--arg", "networkExpr", deploymentPath,
+		"--eval", "--strict", "-A", attribute}
```

Comment:
I think I'd like to "start" from the `nodes` attribute here, so that the user-facing API doesn't expose the implementations details of `eval-machines.nix` -- ie. the user should specify `morph eval foo.nix $nodename.config.networking.hostName` not `morph eval foo.nix nodes.$nodename.config.networking.hostName`